translations/tgl.po

```diff
@@ -68,11 +68,11 @@ msgstr "Mga level ng komunidad"
 
 #: ../client/application/game/ui/shared_strings.cpp:23
 msgid "play count: %s"
-msgstr "Ilang beses nilaro?: %s"
+msgstr "Ilang beses naglaro?: %s"
```

Comment:
Is the question mark here supposed to be before the colon?

Reply:
yes

Reply:
Alright, I'm merging.